### PR TITLE
video_ffmpeg add df.sanitize_filename()

### PR DESCRIPTION
**Pull request by supertobi**
I'm not 100% sure, but I think we need the df.sanitize_filename() here.

### CODE REVIEW

**Comment by supertobi** [comment]
@wpferguson and @BzKevin could you please review this pr?

**Comment by theres** [comment]
yes, we need this

**Comment by wpferguson** [comment]
@supertobi could you add df.sanitize_filename() to the df.mkdir() and df.rmdir() arguments also?  I'll clean up the library elements to make them consistent with the other library functions.

**Comment by supertobi** [comment]
please review

File: contrib/video_ffmpeg.lua [hunk 1]
```diff
@@ -424,7 +424,7 @@ local function finalize_export(storage, images_table, extra_data)
     
     dt.print(_("prepare merge process"))
     
-    local result = df.mkdir(tmp_dir)
+    local result = df.mkdir(df.sanitize_filename())
```
Comment (theres) [inline contrib/video_ffmpeg.lua]: you forgot about an argument

Comment (supertobi) [reply contrib/video_ffmpeg.lua]: arrrggg
fixed

**Pull request merged**